graphql/src/main/java/io/micronaut/configuration/graphql/ws/apollo/GraphQLApolloWsConfiguration.java

```diff
@@ -106,6 +126,15 @@ public boolean isKeepAliveEnabled() {
         return keepAliveEnabled;
     }
 
+    /**
+     * Sets whether GraphQL websocket keep alive is enabled.
```

Comment:
make sure the javadoc which goes into the configuration reference documentation outputs the default value